lib/Sema/PlaygroundTransform.cpp

```diff
@@ -786,6 +799,14 @@ class Instrumenter : InstrumenterBase {
     return std::make_pair(PBD, VD);
   }
 
+  bool shouldLogType(Type Ty) {
+    // Don't try to log ~Copyable types, as we can't pass them to the generic logging functions yet.
+    if (Ty->isNoncopyable()) {
```

Comment:
You can’t pass interface types to isNoncopyable(), it will assert once the noncopyable generics flag is on.